src/SFA.DAS.ApplyService.InternalApi/Controllers/FinancialController.cs

```diff
@@ -65,6 +71,26 @@ public async Task<IActionResult> RecordGrade(Guid applicationId, [FromBody] Fina
             return Ok();
         }
 
+
+        [HttpPost("Clarification/Applications/{applicationId}/Upload")]
+        public async Task<IActionResult> UploadClarificationFile(Guid applicationId, [FromForm] UploadClarificationFileCommand command)
+        {
+            if (Request.Form.Files != null && Request.Form.Files.Any())
+            {
+                var clarificationFileName = Request.Form.Files.First().FileName;
+                var uploadedSuccessfully = await _fileStorageService.UploadFiles(applicationId, 0, 0, "0", Request.Form.Files, ContainerType.Financial, new CancellationToken());
```

Comment:
The sequence number does not match up  (it should be `RoatpWorkflowSequenceIds.FinancialEvidence`)

Also, you're hardcoding "0" as the pageId which isn't quite right. It should be a constant somewhere and named something like 'FinancialEvidence' (like what we do in Gateway as it will has to follow the same pattern when we do clarification for that journey)

Reply:
agree